README.rst

```diff
@@ -44,13 +44,14 @@ report. To run them all inside the docker image, run ``docker-compose
 run web tox``, or if you are running locally, after installing the
 requirements file, just run ``tox``.
 
+
 Continuous Testing
 ~~~~~~~~~~~~~~~~~~
 
-If you want test to run on file changes, the
-``test_requirements.txt`` adds pytest-watcher, which can be started with
-``ptw``. This unfortunately will not work well in the docker environment
-because the file events it uses are fired on the host OS, and not the
-docker OS. I have added an
-`issue<https://github.com/joeyespo/pytest-watch/issues/9>`_ to the
-`pytest-watch repo<https://github.com/joeyespo/pytest-watch>`_
+If you want test to run on file changes, the ``test_requirements.txt``
+adds pytest-watcher, which can be started with:
+``docker-compose run web ptw --poll``
+For additional options like having it say "passed"
+out loud, or sending desktop notifications for failures see the
+`README<https://github.com/joeyespo/pytest-watch/blob/master/README.md>`_.
```

Comment:
Good catch, they are restructured text links, and they need a space between the name and reference.  This should be fixed.  The markdown link format you have there won't work with RST unfortunately, and I always get them mixed up a bit.